lib/vagrant-proxmox/action/create_vm.rb

```diff
@@ -101,25 +101,25 @@ def create_params_lxc(config, env, vm_id)
             password: 'vagrant',
             rootfs: "#{config.vm_storage}:#{config.vm_disk_size}",
             memory: config.vm_memory,
-            description: desc,
-            cmode: config.lxc_cmode.to_s,
-            cpulimit: config.lxc_cpulimit,
-            cpuunits: config.lxc_cpuunits,
-            swap: config.lxc_swap,
-            tty: config.lxc_tty,
-            pool: config.pool
+              description: desc,
+              cmode: config.lxc_cmode.to_s,
+              cpulimit: config.lxc_cpulimit,
+              cpuunits: config.lxc_cpuunits,
+              swap: config.lxc_swap,
+              tty: config.lxc_tty,
+              pool: config.pool
           }.tap do |params|
             params['ssh-public-keys'] = config.lxc_ssh_public_keys
             params[:nameserver] = config.lxc_nameserver.to_s\
-                                    if config.lxc_nameserver
-            params[:onboot] = get_rest_boolean(config.lxc_onboot)
-            params[:protection] = get_rest_boolean(config.lxc_protection)
-            params[:console] = get_rest_boolean(config.lxc_console)
-            add_lxc_network_config(env, params)
-            add_lxc_mount_points(env, config, params)
+              if config.lxc_nameserver
+                params[:onboot] = get_rest_boolean(config.lxc_onboot)
+                params[:protection] = get_rest_boolean(config.lxc_protection)
+                params[:console] = get_rest_boolean(config.lxc_console)
+                add_lxc_network_config(env, params)
+                add_lxc_mount_points(env, config, params)
```

Comment:
And here it seemed to have got confused by `\` at :113. The indent should stay as it was. [The usage of `\` is ugly, but I'm not the one to blame here ;-).]